### PR TITLE
Pedagogy Guide

### PR DESCRIPTION
**Pull request by kseniyausovich**
Intro, Links, Connectors+Modules

### CODE REVIEW

File: content/v2/pedagogy/connectors.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!-- #region -->
+# Connector courses
+
+
+Connectors are two-unit (usually) courses which give students an in-depth introduction into the use of the data science toolkit and concepts in a particular field. Connector courses vary every semester, but are usually within the domain of social sciences. Connectors are not an official part of Data 8, though they play a crucial role in the data science education ecosystem with the DSEP (Data Science Education Program). Students are encouraged to take them together with Data 8 or a semester after they take Data 8.
```
Comment (alexnakagawa) [inline content/v2/pedagogy/connectors.md]:
```suggestion
Connectors are one or two-unit courses which give students an in-depth introduction into the use of the data science toolkit and concepts in a particular field. Connector courses vary every semester, but are usually within the domain of social sciences. Connectors are not an official part of Data 8, though they play a crucial role in the data science education ecosystem with the DSEP (Data Science Education Program). Students are encouraged to take them together with Data 8 or a semester after they take Data 8.
```

File: content/v2/pedagogy/connectors.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+For more information about connector courses, check out [The Data Science Connector Courses Page](https://data.berkeley.edu/education/connectors)
+
+
+
+# Modules 
+
+Modules are short explorations into data science that can be implemented into (almost) any class on campus. Modules allow students to explore a data set relevant to their course with the help of the data science toolkit. 
+
+Like connectors, modules vary widely, and can be customized to fit each instructor's need and objective for the course. A module might include one or two lectures on analyzing course-relevant data and learning how to interpret the graphs, or they can be a set of labs in which students will learn the basics of programming and statistics which they can apply to their field of studies; some modules can also include a final class project ran in Jupyter Notebooks.
```
Comment (alexnakagawa) [inline content/v2/pedagogy/connectors.md]:
```suggestion
Like connectors, modules vary widely, and can be customized to fit each instructor's need and objective for the course. A module might include one or two lectures on analyzing course-relevant data and learning how to interpret the graphs, or they can be a set of labs in which students will learn the basics of programming and statistics which they can apply to their field of studies. Some modules also include a final class project run completely within Jupyter Notebooks.
```

File: content/v2/pedagogy/course_structure.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Data 8 Course Staff Structure
+## Motivation
```
Comment (alexnakagawa) [inline content/v2/pedagogy/course_structure.md:3]:
```suggestion

## Motivation
```

File: content/v2/pedagogy/course_structure.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+The scale of staffing for these support resources is only possible through the enthusiasm of former students in the class. An undergraduate who has performed well in Data 8 and enjoys the course material can join the course staff as an **academic intern (AI)**. Academic interns assist teaching assistants (TAs) by answering student questions in a weekly lab section for 1 academic Pass/No Pass unit - an end of semester reflection on their experience is also required for credit. Undergraduates who prove to be engaged and talented academic interns are then selected to be tutors for the class. 
+
+**Tutors** assist with grading written responses on assignments, holding office hours, and conducting small group tutoring sections. Experienced tutors can eventually advance to positions as **Undergraduate Student Instructors (uGSIs)**, holding a weekly lab section and helping out with various pedagogical and logistical aspects of running the course. This hierarchical course staff structure is enabled by the careful leadership of Head GSIs and course instructors.
+
+
+You can read more about the [Roles and Responsibilities here](roles_data8.md)
```
Comment (alexnakagawa) [inline content/v2/pedagogy/course_structure.md]:
```suggestion
You can read more about the [Roles and Responsibilities here](roles_data8) 
```

Removing the .md extension as the redirect should be an html page.

File: content/v2/pedagogy/intro_pedagogy.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<!-- #region -->
+# The Data 8 Pedagogy Guide
+
+Data 8 is a “The Foundations of Data Science” course taught to first-year students at UC Berkeley. It combines principles and skills from statistics and computer science, such as inference, modeling, hypothesis testing, visualization, and others. It provides a foundation in the many disciplines encompassed by “data science”, and gives students a practical introduction to the field.
```
Comment (alexnakagawa) [inline content/v2/pedagogy/intro_pedagogy.md:4]: It's not just first-year students though. May want to consider removing that part of the sentence.

File: content/v2/pedagogy/intro_pedagogy.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+All of the tools that Data 8 uses are open source, and available for the community to use (either as broader community-run projects, or as Berkeley projects). The course material can be accessed at the following online resources:
+
+- [The Data 8 Course Site](http://data8.org)
+- [The Data 8 Course Textbook](https://www.inferentialthinking.com/chapters/intro)
+
+To explore the guide, select a section to the left!
+
+### Contacting Us
+
+If you woud like to learn more about any of the tools used in Data 8 or are interested in deploying your own data 8 course, please fill out our Data 8 [Instructor Interest form](https://forms.gle/y5aQBi816xDXKe7r7) or shoot us an email at ds-help@berkeley.edu.
```
Comment (alexnakagawa) [inline content/v2/pedagogy/intro_pedagogy.md]:
```suggestion
If you woud like to learn more about any of the tools used in Data 8 or are interested in deploying your own data 8 course, please fill out our Data 8 [Instructor Interest form](https://forms.gle/y5aQBi816xDXKe7r7) or shoot us an email at [ds-help@berkeley.edu](mailto:ds-help@berkeley.edu).
```

File: content/v2/pedagogy/roles_data8.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
+## Roles and Responsibilities
```
Comment (alexnakagawa) [inline content/v2/pedagogy/roles_data8.md]: Turn this into H1? 
```suggestion
# Roles and Responsibilities
```

File: content/v2/pedagogy/roles_data8.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+Each semester, around 45 teaching assistants (TAs), 45 tutors, and 150 academic interns (AIs) are hired to be part of Data 8 course staff. There are differing levels of hourly commitments, roles, and responsibilities both by and within roles depending on the staff member’s experience and ability to contribute to course staff.
+
+### Academic Intern (AIs)
+
+##### Lab assisting (2 hrs/week)
+- 10 out of the 13 weeks (20 hours) → required for P
```
Comment (alexnakagawa) [inline content/v2/pedagogy/roles_data8.md:7]: Clarify what the P means.

File: content/v2/pedagogy/useful_links.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+
+
+
+
+- To find out which schools have either already adopted or are in the process of adopting Data 8, follow [this link](https://data.berkeley.edu/academics/resources/data-science-education-resources/partner-institutions) 
+
+
+
+- To see other schools that have a Data Science Program, follow [this link](https://docs.google.com/spreadsheets/d/1ycAN3V9O4lg8MwczpTj8kDgNts7vA61R23LwF1atvRw/edit#gid=1343969407)
```
Comment (alexnakagawa) [inline content/v2/pedagogy/useful_links.md:12]: Is this a valid link? it breaks for me.

**Review by alexnakagawa — CHANGES_REQUESTED** [review]

File: content/v2/pedagogy/roles_data8.md (hunk 8 above)
Comment (kseniyausovich) [reply content/v2/pedagogy/roles_data8.md:7]: Oh, it's for "Pass". Should I push the corrections and submit a new PR?

File: content/v2/pedagogy/intro_pedagogy.md (hunk 5 above)
Comment (kseniyausovich) [reply content/v2/pedagogy/intro_pedagogy.md:4]: Well, anyone can take it, but it is tailored towards first-year students as it is a lower-div. requirement in quite a few majors

File: content/v2/pedagogy/useful_links.md (hunk 9 above)
Comment (kseniyausovich) [reply content/v2/pedagogy/useful_links.md:12]: works for me

File: content/v2/pedagogy/intro_pedagogy.md (hunk 5 above)
Comment (alexnakagawa) [reply content/v2/pedagogy/intro_pedagogy.md:4]: Ok, you can leave it then.

File: content/v2/pedagogy/roles_data8.md (hunk 8 above)
Comment (alexnakagawa) [reply content/v2/pedagogy/roles_data8.md:7]: Yeah, push corrections to this PR, no need to submit a new one.

**Pull request merged**